### ISSUE TITLE
elasticsearch-Debian.yml is not working for install ELK 7+ versions without using repository

### ISSUE THREAD

**Issue by rubarclk** [elastic/ansible-elasticsearch#758]
-I wanted to download elasticsearch 7.10.2 version without using repository, the url used in the code block below is missing;

```
- name: Debian - Download elasticsearch from url
  **get_url: url={% if es_custom_package_url is defined %}{{ es_custom_package_url }}{% else %}{{ es_package_url }}-{{ es_version }}.deb{% endif %} dest=/tmp/elasticsearch-{{ es_version }}.deb validate_certs=no**
  when: not es_use_repository
```

- Then I checked elasticsearch manual download and install guide and I noticed that the missing part of url is **-amd64**
- https://www.elastic.co/guide/en/elasticsearch/reference/current/deb.html
- I was able to install elasticsearch when I fixed the code block as follows;

```
- name: Debian - Download elasticsearch from url
  get_url: url={% if es_custom_package_url is defined %}{{ es_custom_package_url }}{% else %}{{ es_package_url }}-{{ es_version }}-amd64.deb{% endif %} dest=/tmp/elasticsearch-{{ es_version }}.deb validate_certs=no
  when: not es_use_repository
```

**Comment by jmlrt** [comment]
Hi @rubarclk,
Thanks for reporting this issue.
Do you think you could provide a PR to fix it?

**Comment by jmlrt** [comment]
fixed by #761
